### DISCUSSION TITLE
Update the default branch under setting tab through script

### DISCUSSION THREAD

**Discussion by surajsvcs1231** [community/community#67582] [Repositories]
### Select Topic Area

General

### Body

I am trying to write bash script which will update the default branch from master to other branch but I am not getting any luck.
The git hub repo is in the private organization. Tried using several curl command but no luck. Waiting for some help. Thanks in Advance

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @surajsvcs1231 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!
